README.rst

```diff
@@ -1156,6 +1156,20 @@ is raised and can be rescued and processed. It contains a description
 of the parameter value expectations. Validations can be turned off
 in the configuration file.
 
+Here is an example of how to rescue and process a +ParamMissing+ or
++ParamInvalid+ error from within the ApplicationController.
+
+.. code:: ruby
+  class ApplicationController < ActionController::Base
```

Comment:
this code isn't displayed then the README is rendered.
I think you are only missing a white line after `code:: ruby`
can you try adding one?

Reply:
Good catch. Fixed! Looks better now